### PR TITLE
Add a validation for checking if the scheduler is running

### PR DESCRIPTION
**Pull request by Jellyfrog**
This is just preparation for future use.

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.
- [ ] If my Pull Request makes discovery/polling/yaml changes, I have added/updated [test data](https://docs.librenms.org/Developing/os/Test-Units/).

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: LibreNMS/Validations/Scheduler.php [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+     * Validate this module.
+     * To return ValidationResults, call ok, warn, fail, or result methods on the $validator
+     *
+     * @param  Validator  $validator
+     */
+    public function validate(Validator $validator): void
+    {
+        if (! Cache::has('scheduler_working')) {
+            $validator->fail('Scheduler is not running!',
+                'cp /opt/librenms/librenms.nonroot.cron /etc/cron.d/librenms');
```
Comment (murrant) [inline LibreNMS/Validations/Scheduler.php:40]: This fix won't be correct if they have disabled some of the cron for the dispatcher.

Maybe sed?

Comment (murrant) [reply LibreNMS/Validations/Scheduler.php:40]: or a separate cron file.

Comment (Jellyfrog) [reply LibreNMS/Validations/Scheduler.php:40]: > This fix won't be correct if they have disabled some of the cron for the dispatcher.
> 
> Maybe sed?

TLDR; no hacks with sed plz. The source file contains the row for this scheduler, we can't really do much about users editing files, there is enough of warnings in place for that already?

Since this could/should replace all/most of the cron file in the future, maybe we can do separate ones.
I'll fix.

**Comment by murrant** [comment]
Honestly, we should switch them all to timers.  (but keep the cron file around for non-systemd systems)

**Review by murrant — APPROVED** [review]
